Swagger generated APIs from master

Ropes opened the pull request:
Obviously there have been some changes to code and swagger declarations
which weren't built.

Commands run: `make generate-api` & `make generate-health-api` & `git
add api*`. That's it.

### Backstory 

These changes were noticed when making a modification to the `server.gotmpl` file for #10116. Thus creating a requirement to separate API master code, from what was being changed for the `syscall` package removal.

@aanm asked me to build all of the swagger generated code from master so the diff between #10158 was limited to only the `syscall`->`unix` package changes.

Related: #10116

Signed-off-by: Joshua Roppo joshroppo@gmail.com

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cilium/cilium/10336)
<!-- Reviewable:end -->

joestringer commented on api/v1/health/server/server.go:
```diff
@@ -25,8 +22,11 @@ import (
 	flags "github.com/jessevdk/go-flags"
 	"golang.org/x/net/netutil"
 
+	"github.com/cilium/cilium/api/v1/health/server"
```
I'm surprised at these changes in `api/v1/health/server/server.go` and `api/v1/server/server.go`, only the imports are changing but there's no other changes. Line 290 for instance uses the `x509` package, so this seems like a breaking change. I think we should drop the changes to these two files?

Ropes replied:
Yeah that's a problem.. but we don't even make it that far:
```
make[1]: Leaving directory '/home/josh/go/src/github.com/cilium/cilium/cilium'
make[1]: Entering directory '/home/josh/go/src/github.com/cilium/cilium/daemon'
CGO_ENABLED=0 go build -mod=vendor -ldflags '-X "github.com/cilium/cilium/pkg/version.Version=1.7.90 f411adcbf 2020-02-25T10:34:54-08:00 go version go1.13.6 linux/amd64" -X "github.com/cilium/cilium/pkg/envoy.RequiredEnvoyVersionSHA=c31482c3e49670980c05cafc914320f7949b266f" -X "github.com/cilium/cilium/pkg/datapath/loader.DatapathSHA=5e3b6c150445abcefcc48bd0358fe1b421bb369a"'  -o cilium-agent
import cycle not allowed
package github.com/cilium/cilium/daemon
	imports github.com/cilium/cilium/cilium-health/launch
	imports github.com/cilium/cilium/pkg/health/server
	imports github.com/cilium/cilium/api/v1/health/server
	imports github.com/cilium/cilium/api/v1/health/server
import cycle not allowed
package github.com/cilium/cilium/daemon
	imports github.com/cilium/cilium/api/v1/server
	imports github.com/cilium/cilium/api/v1/server
Makefile:8: recipe for target 'cilium-agent' failed
```

Removing two more files... :grimacing:

joestringer reviewed (changes requested):
Thanks for preparing this! At first glance this seems good, just one comment for the last couple of files below.

Ropes committed:
Swagger generated APIs from master

Obviously there have been some changes to code and swagger declarations
which weren't built.

* Commands run: `make generate-api` & `make generate-health-api`.
* Then removed `api/v1/health/models/health_response.go` on
request of @aanm.
* Unstaged swagger removed `// +k8s:deepcopy-gen=true` lines on
request of @joestringer.
* Removed api/v1/health/server/server.go * api/v1/server/server.go
due to build failure, and odd import changes:
https://github.com/cilium/cilium/pull/10336/files/f411adcbf09a9cf827438a59c189bc8cbe62f54c#r384055153

Related: #10116

Signed-off-by: Joshua Roppo joshroppo@gmail.com

aanm reviewed (approved):
What can we do to not miss these ones? :thinking:
